library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -162,19 +161,27 @@ class GroupTest {
             )
         }
         assertEquals(
-            group.memberAddresses().sorted(),
+            group.members().map { it.inboxId }.sorted(),
             listOf(
-                caro.walletAddress.lowercase(),
-                alix.walletAddress.lowercase(),
-                bo.walletAddress.lowercase()
+                caroClient.inboxId.lowercase(),
```

Comment:
Odd that we have to lowercase `inboxId`s here. Since these always originate from inside libxmtp, can't we just leave the casing alone?

Reply:
Ahh yes no longer needed now that it's not an address